README.md

```diff
@@ -123,6 +123,7 @@ The following languages are currently supported:
 - it: Italian (Italy, Switzerland)
 - ro: Romanian
 - ja: Japanese
+- pt-BR: Portuguese (Brazil)
```

Comment:
Whenever we have a new language, we start exposing it as language without the country specific code.
I'd propose to rename all this from pt-BR to pt and then split it up into pt-BR and pt-PT once someone comes up and suggest specific changes per country. Would you be fine with that? This would also allow Portuguese speakers from other countries than Brazil to automatically see a Portuguese translation.

Reply:
Hi @m-mohr. Thanks for the review. I'm fine with that! No problem, let's rename it to "pt".